tests/Api/StatusTest.php

```diff
@@ -10,6 +10,6 @@
 class StatusTest extends TestCase {
   public function testStatusReturnsOk(): void {
     $response = $this->get('/status');
-    Assert::assertEquals(200, $response->getStatusCode());
+    $this->assertStatus(200, $response);
```

Comment:
it is a bit silly to go though and change all of these just for the sake of changing them.... but whatever

Reply:
I was winding down the night, waiting for dinner so I went through for consistency